BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/IScheduleService.java

```diff
@@ -0,0 +1,46 @@
+package com.brewmes.common.services;
+
+import com.brewmes.common.entities.ScheduledBatch;
+
+import java.util.List;
+
+public interface IScheduleService {
+
+    /**
+     * Adds a batch to the global queue.
+     *
+     * @param speed    Speed of the batch.
+     * @param beerType Beer type of the batch.
+     * @param amount   Amount to produce.
+     * @return the placement of the added {@code ScheduledBatch} in the queue; Returns -1 if the batch could not be placed in the queue.
+     */
+    int addToQueue(ScheduledBatch scheduledBatch);
+
+    /**
+     * Removes a batch from the queue.
+     *
+     * @param scheduleID a {@code ScheduledBatch}'s ID.
+     */
+    void removeFromQueue(String scheduleID);
```

Comment:
```suggestion
     * @return {@code True} if the {@code ScheduledBatch} was successfully removed from the queue; {@code False} if an error occured.
     */
    boolean removeFromQueue(String scheduleID);
```
@Kaszz 
We can change it to this. I'm not sure it would be useful though.